packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/RecordNavigationBar/index.js

```diff
@@ -27,8 +27,8 @@ const propTypes = {
   showRecordInput: PropTypes.bool,
```

Comment:
this commit should have a name that makes sense on its own. there's no way for another person to know what is meant with issues 1 to 3 :-)